addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -1797,104 +1797,115 @@ export class OdooEditor extends EventTarget {
         });
 
         const mainCategories = [
-            { name: 'Format', priority: 30 },
-            { name: 'Structure', priority: 20 },
-            { name: 'Widgets', priority: 10 },
+            { name: 'Structure', priority: 70 },
+            { name: 'Format', priority: 60 },
+            { name: 'Widgets', priority: 30 },
```

Comment:
The first one just adapts the existing code to the new API. The second is actually changing behavior.

Reply:
I wouldn't have defined explicit priorities in a commit just to change them in the next. I think it creates confusion when analyzing the commits, something someone else might do later on in a blaming archeology run. I would have had the commands order be temporarily different during this refactoring that actually introduces the concept of ordering itself before defining the new order. However, some people would argue for the opposite and there is value in that too: the way you did it, we can revert the commit that reorders the stuff while keeping the new api and the original order would still apply. A property that indeed wouldn't hold in my version. It could be alleviated somewhat if we used merge commits, as it would indicate that the resulting state might be invalid when only reverting part of the changes that were contained in the merge commit. Since we mostly avoid merge commits in Odoo unless it is a very big feature, I guess the way you did it is in fact better.